docs/howtos/adding-a-new-component.md

```diff
@@ -200,8 +200,12 @@ Use the Xcode Test Navigator to run your tests and check whether
 they're passing.
 
 ## Distribute your component with `rust-components-swift`
+The Swift source code and generated UniFFI bindings are distributed to consumers (eg: Firefox iOS) through [`rust-components-swift`](https://github.com/mozilla/rust-components-swift).
 
-To distribute your component with `rust-components-swift`, you need to add logic for dynamically generating any swift code to the [`generate.sh` script in the `rust-components-swift` repo](https://github.com/mozilla/rust-components-swift/blob/main/generate.sh). You can use that script to:
```

Comment:
I'm fully eliminating the `generate.sh` script in https://github.com/mozilla/rust-components-swift/pull/120. Right now it's only use case is local builds, but that PR over in rust-component-swift switches local builds to also use the same script as taskcluster